libc/src/__support/high_precision_decimal.h

```diff
@@ -186,19 +188,19 @@ class HighPrecisionDecimal {
   }
 
   // Perform a digitwise binary non-rounding right shift on this value by
-  // shiftAmount. The shiftAmount can't be more than MAX_SHIFT_AMOUNT to prevent
-  // overflow.
-  void right_shift(uint32_t shiftAmount) {
+  // shift_amount. The shift_amount can't be more than MAX_SHIFT_AMOUNT to
```

Comment:
Do you mind putting the clang-tidy fixes to a separate commit next time, since they take a significant part of the diff?

Reply:
I'll be sure to do that next time, thanks for telling me.